web-local/src/lib/redux-store/source/source-selectors.ts

```diff
@@ -23,5 +23,8 @@ export const selectTimestampColumnFromProfileEntity = (
   derivedProfileEntity: DataProfileEntity
 ) => derivedProfileEntity?.profile?.filter(isProfileColumnATimestamp) ?? [];
 
-export const selectTimestampColumnFromModelSchema = (schema: V1StructType) =>
+export const schemaHasTimestampColumn = (schema: V1StructType) =>
+  schema?.fields?.some(isFieldColumnATimestamp);
+
+export const selectTimestampColumnFromSchema = (schema: V1StructType) =>
```

Comment:
We can probably move this out of `redux-store`

Reply:
Ya lemme do that